### DISCUSSION TITLE
Rewriting the installer in C++

### DISCUSSION THREAD

**Discussion by bolt926** [caernarferon/caveman-linux#39] [General]
Trying to rewrite and finish the installer in C++. It should be relatively simple. If you want to write it in Rust feel free to do so but I'm not helping you.

**Comment by spacefault** [comment]
C++ is good, but we all can agree that we need an Assembly installer

**Reply by bolt926** [reply]
Are you trying to make this run _worse_?

**Reply by spacefault** [reply]
We need a fortran installer for the best installer

**Reply by bolt926** [reply]
I don't think that anyone will learn FORTRAN just to make an installer for a random Linux distribution

**Reply by bolt926** [reply]
although what we truly need is an installer in Haskell
so if you are willing to learn Haskell have fun

**Reply by spacefault** [reply]
We need an installer written in HolyC

**Reply by bolt926** [reply]
what about C--

**Reply by spacefault** [reply]
Lets use Emojicode

**Reply by bolt926** [reply]
The true best language is Brainf***

**Reply by spacefault** [reply]
It's actually Malbolge, because my brain seems to run on it for some reason

**Comment by polandlover123** [comment]
can you write it in rust?

**Reply by bolt926** [reply]
No.

**Reply by polandlover123** [reply]
what if i asked nicely

**Reply by bolt926** [reply]
I don't speak Rust
